hack/lib/etcd.sh

```diff
@@ -77,6 +77,10 @@ kube::etcd::start() {
 
   # Start etcd
   ETCD_DIR=${ETCD_DIR:-$(mktemp -d 2>/dev/null || mktemp -d -t test-etcd.XXXXXX)}
+  # Using a ramdisk to get better performance
+  kube::log::info "mount tmpfs ${ETCD_DIR} -t tmpfs"
+  mount tmpfs "${ETCD_DIR}" -t tmpfs
```

Comment:
This requires root which I don't expect integration tests to generally run with. How are arm tests run? Why not just mount  tmpfs in /tmp in the OS image?

Reply:
Hi,
In fact, I have mounted tmpfs into /tmp in my arm test environment.
And then the test became stable now.

I agree that the modification is not so general.
Otherwise, can I just keep the tips and delete the code?